src/sas/qtgui/Utilities/DocViewWidget.py

```diff
@@ -6,6 +6,7 @@
 from typing import Optional
 
 from PySide6 import QtCore, QtWidgets, QtWebEngineCore
+from PySide6.QtGui import QCloseEvent
```

Comment:
There is now a merge conflict here due to the linting work. I think this is ready to be merged once the conflict is resolved.